### DISCUSSION TITLE
Will texter update to PM4?

### DISCUSSION THREAD

**Discussion by Atara6431** [fuyutsuki/Texter#118] [Q&A] [answered]
I  see  texter  open  to  public  again, maybe  can release  a  preliminary  version  first.

**Comment by fuyutsuki** [comment, marked as answer]
Yes, it will be released today or tomorrow in JST (GMT+0900).

**Reply by Atara6431** [reply]
nice!nice!nice!

**Reply by fuyutsuki** [reply]
Now available: [4.1.0](https://github.com/fuyutsuki/Texter/releases/tag/release%2Fv4.1.0)

**Comment by Atara6431** [comment]
brilliant to update PM5

**Reply by Atara6431** [reply]
！
